src/main/java/io/reactivex/observers/ResourceCompletableObserver.java

```diff
@@ -122,8 +122,7 @@ public final void dispose() {
     }
 
     /**
-     * Returns true if this ResourceObserver has been disposed/cancelled.
-     * @return true if this ResourceObserver has been disposed/cancelled
+     * {@inheritDoc}
```

Comment:
If we leave the comment out, the JavaDoc/IDE will happily use the parent docs, such as this: http://reactivex.io/RxJava/2.x/javadoc/io/reactivex/observers/ResourceObserver.html#onSubscribe-io.reactivex.disposables.Disposable-